packages/slice-machine/lib/builders/SliceBuilder/Header/index.tsx

```diff
@@ -130,11 +128,6 @@ const Header: React.FC<{
           />
         </Flex>
       </Box>
-      <MetaData
```

Comment:
I didn't saw anything to open that modal, i'm guessing it was unused, do you confirm ?

Reply:
Yes exactly, it's an abandoned feature I think :)